src/components/NewMovie/NewMovie.tsx

```diff
@@ -1,7 +1,7 @@
 import { useState } from 'react';
 
 type Props = {
-  onAdd: any,
+  onAdd: (movies:Movie) => void,
```

Comment:
```suggestion
  onAdd: (movie:Movie) => void,
```
Это же один фильм, а не много